src/assets/scss/material.orcid.overwrites.scss/button.scss

```diff
@@ -23,8 +23,36 @@ button.mat-button-wrap-text-with-icon {
 
   a {
     position: absolute;
-    left: 5%;
-    top: 20%;
+    left: 20px;
+    top: 5px;
+    img {
+      width: 24px;
+    }
+  }
+
+  .columns-4 & {
+    span {
+      display: inline-block;
+      width: 180px;
+      margin: auto;
+      word-break: break-word;
+      white-space: normal;
+      line-height: 24px;
+      min-height: 36px;
+    }
+  }
+}
+
+[dir='rtl'] button.mat-button-wrap-text-with-icon {
+  width: 100%;
+  border: solid 1px rgba(0, 0, 0, 0.12);
```

Comment:
The colors should be declared on the `_button-theme.scss` file to reuse the palette colors, 
We have an open issue about this https://github.com/ORCID/orcid-angular/issues/219

Reply:
I can see this is just duplicated from the non-rtl CSS so you can just delete this line for now

Reply:
Ok